Spanner/src/Connection/Grpc.php

```diff
@@ -120,11 +122,25 @@ class Grpc implements ConnectionInterface
      */
     private $longRunningGrpcClients;
 
+    private function enableConnectionManagement($conf_path)
+    {
+        // TODO(ddyihai): move this function to GrpcTrait, if we are going
+        // to enable the grpc-gcp library for other apis.
+        $conf = new ApiConfig();
+        $conf->mergeFromJsonString($string = file_get_contents($conf_path));
+        $config = new Config($conf);
```

Comment:
The [first parameter](https://github.com/GoogleCloudPlatform/grpc-gcp-php/blob/master/src/Config.php#L37) when constructing a `Config` seems to be the hostname, is it correct to pass in the configuration here?

Reply:
Done. Sorry about that. I am wondering does $config contain host name information? If so, pass $config['hostName'] seems better.

Reply:
No worries! I don't think the handwritten `SpannerClient` where this config is passed from has that information, but it is available [in the GAPIC quite readily](https://github.com/GoogleCloudPlatform/google-cloud-php/blob/master/Spanner/src/V1/Gapic/SpannerGapicClient.php#L99-L107).